addons/mail/static/src/components/notification_group/notification_group.js

```diff
@@ -61,21 +61,7 @@ class NotificationGroup extends Component {
             // handled in `_onClickMarkAsRead`
```

Comment:
can't this be converted to be using event handled system and be handled fully in model? (commit:  Move onClickGroup and onClickMarkAsRead)